checkmate/db.py

```diff
@@ -114,11 +113,6 @@ def _session(request):  # pragma: no cover
     # always closed.
     @request.add_finished_callback
     def close_the_sqlalchemy_session(_request):
-        connections = (
-            session.transaction._connections  # pylint:disable=protected-access
-        )
-        if len(connections) > 1:
-            LOG.warning("closing an unclosed DB session")
```

Comment:
The 'transaction' attribute isn't there in the new version. I'm not exactly what the hint we are in this situation would be now. We don't need to actually check this, as it's only for the warning, so I've removed it.